src/Sylius/Behat/Page/Admin/Order/ShowPage.php

```diff
@@ -395,6 +400,7 @@ protected function getDefinedElements(): array
             'promotion_total' => '#promotion-total',
             'resend_order_confirmation_email' => '[data-test-resend-order-confirmation-email]',
             'resend_shipment_confirmation_email' => '[data-test-resend-shipment-confirmation-email]',
+            'shipment_shipping_at_date' => '#sylius-shipments .shipped-at-date',
```

Comment:
Where are data_test_attributes?

Reply:
nowhere 🗡 , for now we are using attributes in a shop, but here we have admin :)

Reply:
ok 👍